tf_encrypted/operations/secure_random/secure_random.py

```diff
@@ -0,0 +1,47 @@
+import tensorflow as tf
+import tf_encrypted as tfe
+from tensorflow.python.framework import ops
+from tensorflow.python.framework import dtypes
+import os
+
+dirname = os.path.dirname(tfe.__file__)
+shared_object = dirname + '/operations/secure_random/secure_random_module.so'
+
+
+def secure_random(shape, minval=0, maxval=None, dtype=tf.int32, seed=None, name=None):
+    """
+    Returns random numbers securely.
+
+    .. code-block:: python
+
+        x = secure_random([2, 2], minval=-1000, maxval=1000)
+
+    :param list shape: Shape of the random tensor.
+    :param int minval: Minimum value to return, inclusive.
+    :param int maxval: Maximum value to return, exclusive.
+    :param dtype: Data type of the return random values. Either int32 or int64.
+    :param tf.Tensor seed: The seed to be used when generating the random numbers.
+    :param str name:
+
+    :rtype: tf.Tensor
+    """
+
+    secure_random_module = tf.load_op_library(shared_object)
```

Comment:
could we do this once and for all, instead of each time `secure_random` is called?